### PR TITLE
Release 0.8.0

### PR DESCRIPTION
**Pull request by peterbourgon**
Please test.

### CODE REVIEW

**Comment by tomwilkie** [comment]
Does this need a ```make static```?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,34 @@
...
+New features:
+- Show message in the UI when topologies exceed size limits.
+  [#474](https://github.com/weaveworks/scope/issues/474)
+- Provide container image information in detail pane for containers.
+  [#398](https://github.com/weaveworks/scope/issues/398)
+- When filtering out system containers, also filter out pseudo nodes, if they were only connected to system containers.
+  [#483](https://github.com/weaveworks/scope/issues/483)
+- Show number of filtered nodes in status pane.
+  [#509](https://github.com/weaveworks/scope/issues/509)
+- Improve unit and integration test coverage.
```
Comment (tomwilkie) [inline CHANGELOG.md]: We haven't typically mentioned testing improvements in changelogs, just user visible changes.

**Comment by tomwilkie** [comment]
With the latest change ("Don't allow lagging report generation to prevent report publication") its an LGTM from me.

**Comment by paulbellamy** [comment]
Aside from the confusing message on applications views, it LGTM from me.

**Comment by 2opremio** [comment]
LGTM (tested it on ECS)

**Comment by 2opremio** [comment]
LGTM

**Comment by 2opremio** [comment]
sorry, sloppy fingers

**Comment by davkal** [comment]
LGTM

**Pull request merged**